scania-digital-design-system/tegel
# feat(table): prefix change

**Describe pull-request**  
Changing prefix for table component and its sub-components

**Solving issue**  
Fixes: DTS-1633

**How to test**  
1. Go to the Storybook link below
2. I suggest going commit by commit as I did changes in sequences 
3. Check that component works and looks as in the main branch of Storybook
4. Check if in the docs you can see <tds- as a prefix instead of sdds
5. Checkout branch on local and see if you can find any traces of sdds in table component folder.

**Suggested test steps**

**Additional context**  

1. Expandable story is dependent on button prefix change
